src/acp/rule.test.ts

```diff
@@ -895,10 +899,11 @@ describe("setAgent", () => {
     ).toBe(true);
   });
 
-  it("does not overwrite public and authenticated agents", () => {
+  it("does not overwrite public, authenticated and creator agents", () => {
```

Comment:
Good catch !

Reply:
Yeah, I'm a bit worried that it needed catching, and that people will be using outdated version of the library as more "special" Agents are added to the spec, but it is what it is :/